src/Domain/Regulation/Enum/LocationTypeEnum.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Regulation\Enum;
+
+enum LocationTypeEnum: string
```

Comment:
Curieusement cet enum s'appelle LocationType alors qu'ailleurs on parle de RoadType. Je me souviens que j'avais demandé si on n'uniformiserait pas tout à "location type" et qu'on a décidé que non pour l'instant. Donc mettre cet enum en cohérence ?

```suggestion
enum RoadTypeEnum: string
```